docs/src/main/sphinx/release/release-468.md

```diff
@@ -0,0 +1,32 @@
+# Release 468 (11 Dec 2024)
+
+## General
+
+* Add experimental support for Python user-defined functions. ({issue}`24378`)
+
+## Hive connector
+
+* Deactivate bucket execution when not useful in query processing. ({issue}`23432`)
```

Comment:
This needs to be framed in user-facing impact. Does this reduce CPU/memory utilization?

Reply:
@dain ?